todo/src/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import './Style.css';
+import App from './App';
+import reducer from './reducers';
+
+let persistedState = localStorage.getItem('reduxState')
```

Comment:
Good job on getting local storage working. You and Chaya are the first times I've seen this done.